knimin/lib/data_access.py

```diff
@@ -2563,6 +2562,30 @@ def ut_get_arbitrary_handout_barcode(self, is_AGP=True):
             raise ValueError('No handout barcodes found.')
         return info[0]
 
+    def ut_get_arbitrary_non_ag_barcode(self):
+        """Returns an artibtrarily chosen non-AG barcode
+        For unit testing only!
+
+        Returns
+        -------
+        str: barcode
+            Example: '000001000'
+
+        Raises
+        ------
+        ValueError
+            If no non-AG barcode can be found in the DB.
+        """
+        sql = """SELECT barcode FROM barcodes.project_barcode
+                    WHERE project_id != 1 AND barcode != '000000001'
+                 EXCEPT
+                 SELECT barcode FROM barcodes.project_barcode
+                    WHERE project_id = 1;"""
```

Comment:
The problem is that a given barcode can belong to 2 projects, so I want to make sure that it is not part of the AG project.